pallets/automation-time/src/mock.rs

```diff
@@ -598,6 +589,27 @@ pub fn last_event() -> RuntimeEvent {
 	events().pop().unwrap()
 }
 
+pub fn last_task_id() -> TaskIdV2 {
+	get_task_ids_from_events()
+		.last()
+		.expect("task id is missing from event. maybe job schedule failed")
```

Comment:
This will fail only if the return of `get_task_ids_from_events()`, right? If so, the message should be `Unable to find a task_id from the existing TaskScheduled events`.